pact/ledger.pact

```diff
@@ -315,21 +377,25 @@
 
     (with-default-read ledger (key id account)
       { "balance" : -1.0, "guard" : guard }
```

Comment:
Our use of `-1.0` also emits `RECONCILE` of the receiver as `-1.0`,  so use 0.0 ? Does 0.0 make any difference?

Reply:
good catch